crates/common/tedge_config/src/system_services/managers/config.rs

```diff
@@ -124,7 +124,7 @@ impl SystemConfig {
                 Ok(config)
             }
             Err(_) => {
-                println!("The system config file '{}' doesn't exist. Use '/bin/systemctl' as a service manager.\n", config_path);
+                println!("The system config file '{}' doesn't exist. Using '/bin/systemctl' as the service manager by default.\n", config_path);
```

Comment:
Minor tweak which makes the line slightly shorter as well (and removing the trailing full-stop, as that is not needed)

```suggestion
                println!("The system config file '{}' doesn't exist. Using '/bin/systemctl' as the default service manager\n", config_path);
```